sites/bizbash/components/directory-search/index.marko

```diff
@@ -16,66 +16,76 @@ $ const searchParams = {
   pageNumber: page,
   size: input.pageSize,
   phrase,
+  minimumCapacity,
   facets: [
     { field: 'market', value: marketId },
     { field: 'topic', value: topicId },
   ]
 };
-
-<div class="row">
-  <div class="col mb-block">
-    <form class="directory-search">
-      <input name="phrase" type="search" class="form-control directory-search__input" placeholder="Enter search phrase..." value=phrase>
-      <if(marketId)><input name="marketId" type="hidden" value=marketId></if>
-      <if(topicId)><input name="topicId" type="hidden" value=topicId></if>
-      <button type="submit" class="btn btn-primary directory-search__button">Search</button>
-    </form>
+<form id="directory-search">
```

Comment:
I don't believe that the `<form>` needs to wrap the facets and the search results. Can it be changed to only wrap the search field and the venue capacity value?